editor/audio_stream_preview.cpp

```diff
@@ -129,7 +129,7 @@ void AudioStreamPreviewGenerator::_preview_thread(void *p_preview) {
 			float max = -1000;
 			float min = 1000;
 			int from = uint64_t(i) * to_read / to_write;
-			int to = uint64_t(i + 1) * to_read / to_write;
+			int to = (uint64_t(i) + 1) * to_read / to_write;
```

Comment:
I'm not sure why `i` is being cast to `uint64_t` at all here to be downcasted back to `int` anyway. Might be worth changing all above variables to be `uint64_t`, or drop that cast if not relevant.

Reply:
I think the cast is made so that the multiplication by `to_read` doesn't overflow. While floating-point might be used for the fraction `to_read / to_write`, it is slightly slower than this.

Reply:
Ah that makes sense, so I guess this change is fine as is.